brave/brave-core
# Get back `Tabs from other devices` history section;

This reverts some changes from https://github.com/brave/brave-core/pull/212 and adds `More...` item into History menu for devices which have tabs number in session more than 4.


Resolves https://github.com/brave/brave-browser/issues/10473

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Create sync chain with 2 devices (A and B)
2. Enable on devices `History` and `Open Tabs` sync types
3. On device A open 3 tabs, excluding internal pages (brave://****), like: google.com, apple.com, microsoft.com
4. On device B open 5 tabs, excluding internal pages (brave://****), like: cnn.com, espn.com, nytimes.com, reddit.com, ebay.com
5. On devices A and B open History page (Ctrl+H or `brave://history`) ensure there is a section `Tabs from other devices` (you may need to extend width if the width of a window is small)
6. On device A open app menu, then History sub menu, ensure you can see tabs from device B and `More...` item which leads to `brave://history/syncedTabs`
7. On device B open app menu, then History sub menu, ensure you can see 3 tabs from device A and there is no `More...` item 

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
